warehouse/query-core/src/test/java/datawave/test/JexlNodeAssert.java

```diff
@@ -189,11 +189,101 @@ public JexlNodeAssert hasNumChildren(int numChildren) {
      *            the child
      * @return the new {@link JexlNodeAssert} for the child
      */
-    public JexlNodeAssert child(int child) {
+    public JexlNodeAssert assertChild(int child) {
         isNotNull();
         return assertThat(actual.jjtGetChild(child));
     }
     
+    /**
+     * Returns a new {@link JexlNodeAssert} that will perform assertions on the child at index 0.
+     *
+     * @return the new {@link JexlNodeAssert} for the child
+     */
+    public JexlNodeAssert assertFirstChild() {
```

Comment:
I guess these methods are convenient, but I'd be in favor of just having assertChild(int child) as that seems pretty clear and straightforward to me.

Reply:
I don't feel super strongly one way or the other, so I'm fine with removing the convenience methods.